tests/selenium.go

```diff
@@ -554,6 +621,24 @@ func (s *seleniumHelper) stopVideoRecord() {
 	s.currVideoName = ""
 }
 
+func (s *seleniumHelper) switchToGremlinFilteredQueryTopology(topologyFilterQuery string) error {
```

Comment:
not used

Reply:
It was in use here
https://github.com/skydive-project/skydive/pull/1481/commits/36a1aa5453204330c8e4ecaedc7cfefacc354c5a#diff-afb2b0460e46d8b8ce3da9f8b974b27eR147